src/IntelliTect.Coalesce.CodeGeneration/Generation/Generators/Generator.cs

```diff
@@ -51,7 +52,8 @@ public virtual void Configure(JObject obj)
             foreach (var configProp in properties)
             {
                 var propType = configProp.Type.TypeInfo;
-                configProp.PropertyInfo.SetValue(this, obj.ToObject(propType));
+                JToken value = obj.GetValue(configProp.Name, StringComparison.OrdinalIgnoreCase);
```

Comment:
This a breaking change to the generator config, but I think this is a better way to do it in the long run. Plus, the generator config isn't actually documented since this is the very first config property excluding "disabled". Props for figuring out that this even existed!

The original idea was that any of your `[GeneratorConfig]` properties would be property bag POCOs, but realistically, there aren't going to be so many configurable properties that they need to all be shoved in a bag. Child objects can still exist when needed.